.github/workflows/release_library_download_pytorch_org.yml

```diff
@@ -0,0 +1,73 @@
+name: Release ecosystem library from test to production.
+
+on:
+  push:
+    branches:
+      - test-promotion
```

Comment:
This will be removed before pushing
Plan it to allow run only from pytorchbot-env